docs/samples/deployments/spark/notebooks/02_transform_presidio_csv.py

```diff
@@ -0,0 +1,105 @@
+# Databricks notebook source
```

Comment:
This seems quite similar to the 01 notebook. I would assume that for the CSV case you show how to anonymize all the columns (or a selected set of columns) in a spark DataFrame. If that's not the case, could these two notebooks be merged somehow? Like have a flag of whether it's csv or text?